stabilizer/src/main/java/com/hazelcast/stabilizer/coordinator/CoordinatorCli.java

```diff
@@ -46,6 +46,10 @@
             "Number of Cluster Client Worker JVM's")
             .withRequiredArg().ofType(Integer.class).defaultsTo(0);
 
+    private final OptionSpec<Boolean> autoCreateHZInstancesSpec  = parser.accepts("autoCreateHzInstances",
```

Comment:
What does this mean when it is set to false? I can get that it doesn't auto create, but what is the purpose? So can you give a usecase when this is needed and how it fits in the tomcat integration issue.